estela-api/core/models.py

```diff
@@ -301,6 +301,10 @@ class SpiderJob(models.Model):
     request_count = models.PositiveBigIntegerField(
         default=0, help_text="The number of requests made by the spider job."
     )
+    proxy_usage_data = models.JSONField(
+        default=dict,
+        help_text="Proxy Usage data.",
+    )
```

Comment:
I think it would be good to add a comment on how the `proxy_usage_data` would look. What fields would this JSON have?

Reply:
Done